contracts/crosschain/utils/ERC7786Attributes.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.27;
+
+import {IERC7786Attributes} from "../../interfaces/IERC7786Attributes.sol";
+
+/// @dev Library of helper to parse/process ERC-7786 attributes
+library ERC7786Attributes {
+    /// @dev Parse the `requestRelay(uint256,uint256,address)` (0x4cbb573a) attribute into its components.
+    function tryDecodeRequestRelay(
+        bytes memory attribute
+    ) internal pure returns (bool success, uint256 value, uint256 gasLimit, address refundRecipient) {
```

Comment:
We're only using the calldata version in this PR. Can we keep only that one? I have a sense we'd never have attributes in memory so I'd also remove the `*Calldata` prefix from the name. (Perhaps we needed parsing from memory when we used to send attributes to the receiver.)